doc/content/verification_and_validation/ver-1dc.md

```diff
@@ -90,6 +90,100 @@ The analytical solution for the permeation transient is compared with TMAP8 resu
+    \int_\Omega \psi_i \frac{\partial C_{T_i}}{\partial t} \, d\Omega - \int_\Omega \psi_i \alpha_t^i \frac{C_{T_i}^{\text{empty}} C_M}{N \cdot \text{trap\_per\_free}} \, d\Omega + \int_\Omega \psi_i \alpha_r^i C_{T_i} \, d\Omega = 0.
+\end{equation}
+
+#### Step 4: Integrate by parts and apply the divergence theorem
+
+For the divergence term in [eqn:diffusion_mobile_step3], applying integration by parts and the divergence theorem provides
+\begin{equation}
+    \int_\Omega \psi \nabla \cdot \left( D \nabla C_M \right) \, d\Omega = - \int_\Omega \nabla \psi \cdot \left( D \nabla C_M \right) \, d\Omega + \oint\limits_{\partial \Omega} \psi \left( D \nabla C_M \right) \cdot \mathbf{n} \, d\partial \Omega,
+\end{equation}
+where $\partial \Omega$ is the boundary of the domain and $\mathbf{n}$ is the outward-facing normal vector.
+
+This update term is then substituted back into [eqn:diffusion_mobile_step3], which leads to:
+\begin{equation} \label{eqn:diffusion_mobile_step4}
+    \int_\Omega \psi \frac{\partial C_M}{\partial t} \, d\Omega + \int_\Omega \nabla \psi \cdot \left( D \nabla C_M \right) \, d\Omega - \oint\limits_{\partial \Omega} \psi \left( D \nabla C_M \right) \cdot \mathbf{n} \, d\partial \Omega + \int_\Omega \psi \text{trap\_per\_free} \cdot \sum_{i=1}^{3} \frac{\partial C_{T_i}}{\partial t} \, d\Omega = 0.
+\end{equation}
+
+Since no divergence terms exist in [eqn:trapped_rate_step3], no integration by parts is needed.
+
+#### Step 5: Derive the final weak form in inner product notation
+
+[eqn:diffusion_mobile_step4] and [eqn:trapped_rate_step3] can be expressed in inner product notation, where $\langle a, b \rangle = \int_\Omega a b \, d\Omega$.
+
+[eqn:diffusion_mobile_step4] becomes
+\begin{equation}
+    \langle \psi, \frac{\partial C_M}{\partial t} \rangle + \langle \nabla \psi, D \nabla C_M \rangle - \langle \psi, \text{trap\_per\_free} \cdot \sum_{i=1}^{3} \frac{\partial C_{T_i}}{\partial t} \rangle + \int_{\partial \Omega} \psi \left( D \nabla C_M \right) \cdot \mathbf{n} \, d\Gamma = 0,
+\end{equation}
+and, for $i=1$, $i=2$, and $i=3$, [eqn:trapped_rate_step3] becomes
+\begin{equation}
+    \langle \psi_i, \frac{\partial C_{T_i}}{\partial t} \rangle - \langle \psi_i, \alpha_t^i \frac{C_{T_i}^{\text{empty}} C_M}{N \cdot \text{trap\_per\_free}} \rangle + \langle \psi_i, \alpha_r^i C_{T_i} \rangle = 0.
+\end{equation}
+These weak forms of the equations can now be used for finite element analysis and MMS.
+
+### Application of the MMS
+
+To perform MMS we select a smoothly varying sinusoidal spatial solution for the mobile and trapped species. In order to prevent temporal error from polluting the spatial error we pick temporal dependence that can be exactly represented by the time integrator we choose. In this MMS case we use implicit or backwards Euler which is first order accurate; consequently we choose a linear dependence on time. For our 1D case this leads us to select the following MMS solution for the mobile species:
```

Comment:
```suggestion
To perform MMS, we select a smoothly-varying sinusoidal spatial solution for the mobile and trapped species. In order to prevent temporal error from polluting the spatial error, we pick temporal dependence that can be exactly represented by the time integrator we choose. In this MMS case, we use implicit or backwards Euler which is first-order accurate; consequently, we choose a linear dependence on time. For our 1D case, this leads us to select the following MMS solution for the mobile species:
```